### DISCUSSION TITLE
AudioEngine HD6 codes

### DISCUSSION THREAD

**Discussion by luckman212** [probonopd/irdb#102] [Q&A]
Anyone know of a source for discrete codes for the [AudioEngine HD6](https://audioengine.com/shop/wirelessspeakers/hd6-wireless-speakers/) ? I would love to know if there's a discrete "unmute" or "mute" code. I've still got my original remote, so I was able to learn the mute button which is just a toggle, and does not always work reliably. So, a discrete "mute on" code that could be fired 5 times in a row would be super helpful.

**Comment by luckman212** [comment]
I opened a support case with Audioengine on this. If I get any useful info back I'll share it here.

**Reply by probonopd** [reply]
Thank you very much for sharing @luckman212.

It may also be worth to try out the other functions in the same command set that your remote uses by brute force.

**Comment by ubuesque** [comment]
Hello
Any answer from audioengine or any progress ?

I've HD6 speakers, but my remote is dead.

Does anyone who has a working remote and IR blaster share the codes to replicate the remote PLEASE ?

Or, could you please share here the codes you learned for each button of the remote ?

Thanks in advance
